pkg/sql/schema_changer.go

```diff
@@ -1117,9 +1122,6 @@ func (s *SchemaChangeManager) Start(stopper *stop.Stopper) {
 							// We successfully executed the schema change. Delete it.
 							delete(s.schemaChangers, tableID)
 						}
-						// Advance the execAfter time so that this schema
-						// changer doesn't get called again for a while.
-						sc.execAfter = timeutil.Now().Add(delay)
```

Comment:
This is concerning! What was this doing before if it was not pushing the execAfter ahead 6min. `newTimer` has a timer that fires immediately?